new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleDomainValidation.java

```diff
@@ -41,6 +49,46 @@ public void testCreatingDomain() {
         "Failed to create unique namespace due to ApiException");
     logger.info("Got a new namespace called {0}", namespace);
 
+    // create persistent volume and persistent volume claim
+    final String pvcName = domainUID + "-pvc"; // name of the persistent volume claim
+    final String pvName = domainUID + "-pv"; // name of the persistent volume
+
+    V1ObjectMeta pvcmetadata = new V1ObjectMetaBuilder()
+        .withName(pvcName)
+        .withNamespace(namespace)
+        .build()
+        .putLabelsItem("weblogic.resourceVersion", "domain-v2")
+        .putLabelsItem("weblogic.domainUID", domainUID);
+    V1PersistentVolumeClaimSpec pvcspec = new V1PersistentVolumeClaimSpec()
```

Comment:
Same.  No need for variable here.  Move into position on line 69.

Reply:
corrected it.